app/bundles/CoreBundle/EventListener/ExceptionListener.php

```diff
@@ -45,7 +46,7 @@ public function onKernelException(ExceptionEvent $event, string $eventName = nul
         }
 
         // Check for exceptions we don't want to handle
-        if ($exception instanceof AuthenticationException || $exception instanceof AccessDeniedException || $exception instanceof LogoutException
+        if ($exception instanceof AuthenticationException || $exception instanceof AccessDeniedException || $exception instanceof LazyResponseException || $exception instanceof LogoutException
```

Comment:
When a response is set it gets wrapped in LazyResponseException and then handled by the Security bundle error listener - but because this exception listener is 255 it happened before it. I assume these exceptions are filtered here because of that but it was missing this one to allow the response to work properly and not show a 500